spot-client/config.js

```diff
@@ -124,10 +124,11 @@ window.JitsiMeetSpotConfig = {
      * the jitsi UI.
      */
     XMPP_CONFIG: {
-        bosh: process.env.XMPP_BOSH || '',
+        bosh: process.env.XMPP_BOSH || 'https://beta.meet.jit.si/http-bind',
```

Comment:
Was it intentional to set these to beta?  Might it not make sense for the open source to use meet.jit.si by default instead of beta?